app-scroll-effects/app-scroll-effects-behavior.html

```diff
@@ -285,7 +285,33 @@
   }];
 
   /**
-   * Defines the namespace for the effects
+   * Defines the namespace for the scroll effects.
+   *
+   * The `Polymer.AppScrollEffects` is a helper function that scrolls to a particular 
+   * set of coordinates in a scroll target without scroll effects transitions.
+   *
+   * @param {number} scrollTop 
+   * @param {?HTMLElement} scrollTarget
    */
-  Polymer.AppScrollEffects = Polymer.AppScrollEffects || {};
+  Polymer.AppScrollEffects = function scroll(scrollTop, scrollTarget) {
```

Comment:
I'd suggest we name this function more meaningful like `scrollTo` instead of just `AppScrollEffects`.